app/figures.py

```diff
@@ -107,7 +107,8 @@ def fig10(all_params, years):
                             x='y', title='Cost of electricity generation ($/MWh)')
 
 
-def fig11a(all_params, years):
+def fig11a(all_params, years, land_use):
+    mode_crop_combo = land_use.mode_crop_combo()
```

Comment:
By converting the `land_use` into a class we can decide to only get what we need from it here (`mode_crop_combo`).